src/lsp/protocol.ts

```diff
@@ -601,6 +607,7 @@ export interface AssistCreatingConnectionParams {
   isSonarCloud: boolean;
   serverUrlOrOrganisationKey: string;
   token: string;
+  region?: number;
```

Comment:
Is this type expected here? Should it be the same as the one used in `ConnectionSuggestion`?

Reply:
In fact, the type in ConnectionSuggestion should be a `number` too. I decided to do the conversion completely on the client side.